src/lib/Components/ArtworkGrids/InfiniteScrollGrid.tsx

```diff
@@ -20,7 +18,7 @@ import { isCloseToBottom } from "lib/utils/isCloseToBottom"
 import { ArtistRelayProps } from "./RelayConnections/ArtistForSaleArtworksGrid"
 import { GeneRelayProps } from "./RelayConnections/GeneArtworksGrid"
 
-export const PageSize = 10
+import { PAGE_SIZE } from "lib/data/constants"
```

Comment:
I saw `PAGE_SIZE` being repeated all over so added this new constants file here (unless there's a better location)